jsk_arc2017_common/config/label_names.yaml

```diff
@@ -0,0 +1,43 @@
+label_names:
```

Comment:
Maybe you can use `<rosparam command="load" file="label_names.yaml" ns="label_names" />`.